### ISSUE TITLE
Fallout Tactics: Brotherhood of Steel (38420)

### ISSUE THREAD

**Issue by dreamer** [ValveSoftware/Proton#1945]
# Compatibility Report
- Name of the game with compatibility issues: Fallout Tactics: Brotherhood of Steel
- Steam AppID of the game: 38420
- ProtonDB link with reports: https://www.protondb.com/app/38420

## System Information
- GPU: GTX 770
- Driver/LLVM version: NVIDIA 410.73
- Kernel version: 4.18.16
- Link to full system information report as [Gist](https://gist.github.com/): https://gist.github.com/dreamer/03179db6f82b01e06ec8fa41faf09f00
- Proton version: 3.16-4

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

## Symptoms

Almost the same as #1503 and #1774 - game does not start out of the box. To force game to start, rename `BOS_HR.exe` to `TacticsLauncher.exe`. On first start, intro movie practically does not work (extremely choppy), but that's because game starts in 16bpp mode by default. After changing resolution to native with 32bpp (e.g. "1920 x 1080 x 32") , intro movie works perfectly (as seemingly rest of the game).

**Comment by kisak-valve** [comment]
Hello @dreamer, with the launcher, please add [`PROTON_LOG=1 %command%`](https://github.com/ValveSoftware/Proton#runtime-config-options) to the game's launch options and drag and drop the generated $HOME/steam-$APPID.log into the comment box.

**Comment by dreamer** [comment]
I removed my pfx to get whole install log, it's here: [steam-38420.install.log](https://github.com/ValveSoftware/Proton/files/2568578/steam-38420.install.log)
And then another log when trying to start after installation: [steam-38420.log](https://github.com/ValveSoftware/Proton/files/2568579/steam-38420.log)

Some interesting lines in installation log:

    11495.497:000f:0015:err:winediag:CorIsLatestSvc If this function is called, it is likely the result of a broken .NET installation
    ...
    11495.504:000f:0015:err:eventlog:ReportEventW L".NET Runtime Optimization Service (clr_optimization_v4.0.30319_32) - Tried to start a service that wasn't the latest version of CLR Optimization service. Will shutdown\n"

In log you can see the same problem as in other Fallouts:

    11530.918:0024:0025:err:mscoree:CLRRuntimeInfo_GetRuntimeHost Wine Mono is not installed

So basically .net 4.0 bundled with Fallout Tactics does not install; FT uses this depot: https://steamdb.info/depot/229003/.

Maybe I should create an issue specific to this .net installation? It would cover all 3 older Fallout titles.

**Comment by kisak-valve** [comment]
Thanks, #18 and #1786 should be the current issues covering .net in general.

**Comment by madewokherd** [comment]
This has a launcher that works in wine-mono, so #18 should fix it.

**Comment by dreamer** [comment]
Update after 4.2-3: 
* launcher works, there's no need to rename files any more
* game starts in 16bpp, which hurts performance (especially for intro/cutscenes) - switching to 32bpp fixes the issue
* game is generally playable, works very well
* Steam overlay does not work (just as in Fallout 1 and 2)
* When desktop notification happens (either notification or e.g. when changing volume in Gnome) - some keys on keyboard stop working (e.g. space to switch between team members); problem fixes itself after loading from save (It would be nice to know if similar problem happens in Windows, because developer recommends playing using Win8 in window instead of fullscreen)

**Comment by ranplayer** [comment]
There is a regression in this game when using Proton versions higher than 9.0-4 (tested up to Experimental 2025-12-23) : keyboard input does not work when typing a new character name (that is necessary to start the campaign)

Proton logs: [steam-38420.tar.gz](https://github.com/user-attachments/files/24453675/steam-38420.tar.gz)

Specs:
- OS: Arch
- Kernel: 6.12.61
- GPU: RX 6700 XT (Mesa 25.2.7)
- CPU: Ryzen 7 5700G (8c/16t)
- DE: Plasma 6.5.3 (wayland)
